core/domain/value_generators_domain_test.py

```diff
@@ -29,20 +30,22 @@
 class ValueGeneratorsUnitTests(test_utils.GenericTestBase):
     """Test the value generator registry."""
 
-    def test_value_generator_registry(self):
+    def test_value_generator_registry(self) -> None:
         copier_id = 'Copier'
 
         copier = value_generators_domain.Registry.get_generator_class_by_id(
             copier_id)
-        self.assertEqual(copier().id, copier_id)
+        # Since base class generator is not callable hence copier
+        # function call was replaced.
+        self.assertEqual(copier.id, copier_id)
```

Comment:
I don't understand this change.

Reply:
Since the get_generator_class_by_id was not properly typed, I was getting an error here earlier. Removed it now.